prow/repoowners/repoowners.go

```diff
@@ -736,8 +737,35 @@ func (o *RepoOwners) applyOptionsToPath(path string, opts dirOptions) {
 	}
 }
 
-func (o *RepoOwners) filterCollaborators(toKeep []github.User) *RepoOwners {
-	collabs := sets.New[string]()
+// Extend github team to appproves/reviewers list
+// If appproves/reviewers name contains '/', like 'myorg/developer'
+// It will be recognized as github team.
+func extendGithubTeam(ars sets.String, ghc githubClient) sets.String {
+	newArs := sets.NewString()
+	for ar := range ars {
+		if strings.Contains(ar, "/") {
+			arInfo := strings.Split(ar, "/")
```

Comment:
No reason to do `Contains` and then `Split`, you can split right away and just check the count of splits